core/client/app/templates/setup/three.hbs

```diff
@@ -12,7 +12,8 @@
 sally.sanders@example.com" value=users}}
     </form>
 
-    <button {{action "signin"}} class="btn btn-default btn-lg btn-block {{buttonClass}}">
+    <button {{action 'invite'}} class="btn btn-default btn-lg btn-block {{buttonClass}}">
         {{buttonText}}
     </button>
+    <a {{action 'signin'}} href="#" class="gh-flow-skip">I'll do this later, take me to my blog!</a> 
```

Comment:
You mean on the ember side, or the UI (as in, you don't like a text link down there).

On the ember side, I guess that should just be a regular link now, since signin happened in the previous step.